notebooks/examples/quickstart_in_5_minutes.ipynb

```diff
@@ -142,7 +142,16 @@
      "start_time": "2021-12-08T21:43:44.423627Z"
```

Comment:
<p>How come in the output here we don't see anything?</p><p>Something weird here (on one hand no "nothing found" but on the other hand empty display, without the "traffic light" showing that the drift is minimal / non-existant)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>